cvmfs/nfs_shared_maps.cc

```diff
@@ -206,20 +208,17 @@ bool GetPath(const uint64_t inode, PathString *path) {
   path->Assign(raw_path, strlen(raw_path));
   sqlite3_reset(stmt_get_path_);
   pthread_mutex_unlock(&lock_);
-  atomic_inc64(&dbstat_inode_found_);
+  perf::Inc(n_dbstat_inode_found_);
   return true;
 }
 
 
 string GetStatistics() {
```

Comment:
This comment seems to be misplaced (is for `DownloadManager::GetStatistics()`, right?).  Please see my follow-up comment.  It probably makes sense to keep the method in `DownloadManager`.